src/libraries/ArrayUtil.sol

```diff
@@ -384,6 +478,18 @@ library ArrayUtil {
         return a;
     }
 
+    function populate(
+        int256[] memory a,
+        int256[] memory b,
+        uint256 z
+    ) internal pure returns (int256[] memory) {
+        for (uint256 i = 0; i < a.length; i++) {
+            a[z + i] = b[i];
+        }
+        return a;
+    }
```

Comment:
don't need to return a, because a is updated in-memory

so we can get rid of returns `returns (int256[] memory) ` and `return a`, and just use it as follow:

```solidity
y = new int256[](x.length);
populate(y, x, 0);
```

Reply:
ok will do.